### ISSUE TITLE
Bug report for Unused Schedules

### ISSUE THREAD

**Issue by mldichter** [NREL/EnergyPlus#6743]
Using Energyplus 8.9 and
Output:Diagnostics,
    DisplayUnusedSchedules

for linked idf file, the schedules
AIR VELOCITY SCHEDULE
CLOTHING SCHEDULE
WORK EFFICIENCY SCHEDULE

are listed as unused, but they are referenced by the People object.
People,
    Warehouse Office People, !- Name
    Warehouse Office,        !- Zone or ZoneList Name
    Warehouse BLDG_OCC_SCH,  !- Number of People Schedule Name
    People/Area,             !- Number of People Calculation Method
    ,                        !- Number of People
    0.0210972644167511,      !- People per Zone Floor Area {person/m2}
    ,                        !- Zone Floor Area per Person {m2/person}
    0.3,                     !- Fraction Radiant
    ,                        !- Sensible Heat Fraction
    Warehouse Office Activity Schedule,  !- Activity Level Schedule Name
    ,                        !- Carbon Dioxide Generation Rate {m3/s-W}
    ,                        !- Enable ASHRAE 55 Comfort Warnings
    ,                        !- Mean Radiant Temperature Calculation Type
    ,                        !- Surface Name/Angle Factor List Name
    Work Efficiency Schedule,!- Work Efficiency Schedule Name
    ,                        !- Clothing Insulation Calculation Method
    ,                        !- Clothing Insulation Calculation Method Schedule Name
    Clothing Schedule,       !- Clothing Insulation Schedule Name
    Air Velocity Schedule;   !- Air Velocity Schedule Name

And deleting those three schedules from the idf file results in a severe error.

idf file google drive link
https://drive.google.com/file/d/14ofPMDyjMqCMG7NG3LIofWQiwJAzpL5O/view?usp=sharing

err file google drive link
https://drive.google.com/file/d/1tAXULNflC-yfvLuX_yANoVMoT_E21kRN/view?usp=sharing

**Comment by JasonGlazer** [comment]
Originally reported:

https://unmethours.com/question/31458/bug-report-for-unused-schedules/

**Comment by RKStrand** [comment]
@mldichter @JasonGlazer So, here is what I found while I investigated this issue.  The problem arises because the input file does not choose a thermal comfort model and as a result the schedules are not actually used for anything in the file.  So, in effect, EnergyPlus is trying to tell the user something, but it is not very unclear as to what is being said.  I added a thermal comfort model on the end of the People statement and the unused schedule messages went away.  Technically, there isn't a "problem" but the reality is that this is a bit sloppy and the user can get really confused.  So, here is what I plan to do:

1. Institute a warning message that tells the user if they put in values for things like the work efficiency schedule, clothing schedule, or air velocity but do not pick a thermal comfort model that these schedules will be reported as unused because they have no impact on the results.

2. Add some clarification information to the Input/Output Reference that notes that these schedules are useless if the user does not specific a thermal comfort model.

3. I noticed that the example IDF snippets in the I/O Reference in this section are out of date.  I will fix that as well.
